src/index.ts

```diff
@@ -445,6 +445,10 @@ export class BigQuery extends common.Service {
           value = BigQuery.timestamp(new Date(value * 1000));
           break;
         }
+        case 'GEOGRAPHY': {
```

Comment:
This looks like a feature!  If it is, we should use the PR title `feat(geo): add support for geography` or something like that 😄